server/utils/tans.py

```diff
@@ -1,15 +1,16 @@
 import random
 import string
 
-
-ALLOWED_CHARS = string.ascii_uppercase + string.digits
+import models
+from app import create_app
+from app_config import csrf, db
 
 
-def possible_tans(length: int) -> int:
-    return len(ALLOWED_CHARS) ** length
+ALLOWED_CHARS = string.ascii_uppercase + string.digits
 
 
 class Tan:
+
     value: str = ""
 
     def __init__(self, length_or_value: int | str = 5):
```

Comment:
why dont use two different vars and skip the instance check due to type constraints in the argument and we get rid of a raising TypeError.